File: ruby-pr/53/ruby/code-quality-naming/calculator.rb
class Calculator
  # {fact rule=code-quality-naming@v1.0 defects=1}
  def calc(a, b, op)
    case op
    when '+'
      a + b
    when '-'
      a - b
    when '*'
      a * b
    when '/'
      b != 0 ? a / b : 0
    else
      0
    end
  end
  # {/fact}
end

File: kotlin-pr/50/kotlin/code-quality-naming/Calculator.kt
class Calculator {
    // {fact rule=code-quality-naming@v1.0 defects=1}
    fun calc(a: Double, b: Double, op: String): Double {
        return when (op) {
            "+" -> a + b
            "-" -> a - b
            "*" -> a * b
            "/" -> if (b != 0.0) a / b else 0.0
            else -> 0.0
        }
    }
    // {/fact}
}